coalib/coala_delete_orig.py

```diff
@@ -7,9 +9,14 @@
 from coalib.settings.ConfigurationGathering import get_config_directory
 from coalib.settings.Section import Section
 from coalib.parsing.Globbing import glob_escape
+from coalib.misc.Constants import log_settings_file
 
 
 def main(log_printer=None, section: Section=None):
+    # Configure logging
+    with open(log_settings_file) as fd:
+        logging.config.dictConfig(json.load(fd))
```

Comment:
yeah make it a function :) or a decorator? like `@configure_logging(config_file_path)`?